plugins/scaffolder-backend/report.api.md

```diff
@@ -102,17 +102,16 @@ export interface CreateBuiltInActionsOptions {
   // (undocumented)
   additionalTemplateGlobals?: Record<string, TemplateGlobal_2>;
   auth?: AuthService;
-  catalogClient: CatalogApi;
+  catalogClient: CatalogService;
```

Comment:
Ah, these are gonna be breaking changes that we'd like to avoid at this point. So we'll need to keep the `CatalogApi` in use here for the time being : /

Reply:
Alright, will revert 👍🏼

Reply:
@Rugvip how do we feel about actually shipping the breaking change at the same time as the legacy backend system removal?